WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -45,7 +46,11 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		Observable.FromEventPattern<string>(_qrReader, nameof(_qrReader.InvalidAddressFound))
 			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(args => Message = $"Invalid QR code.")
+			.Subscribe(args =>
+			{
+				ErrorMessage = "The scanned QR code is not a valid Bitcoin address";
```

Comment:
How about this to solve https://github.com/zkSNACKs/WalletWasabi/pull/7516#issuecomment-1065398153

```suggestion
				ErrorMessage = "The QR code is not a valid Bitcoin address";
```